README.md

```diff
@@ -143,12 +143,12 @@ auth    requisite                       pam_deny.so
 
 ## Testing
 
-It is easy to do a quick test to ensure the duress module is working properly.
+It is easy to do a quick test to ensure the duress module is working properly. make sure to replace "user" by your username.
 
 ```bash
 $> mkdir -p ~/.duress
 $> echo '#!/bin/sh
-echo "Hello World"' > ~/.duress/hello.sh
+echo "Hello World" > /home/user/test.txt' > ~/.duress/hello.sh
```

Comment:
What is purpose of this change? Instead of directly getting the "Hello World" echoed to the output, users would now need to read from the file. Also it requires more steps now.

Reply:
@DusanLesan, this was in response to issue #47 which I'm still finding time to replicate and am hoping to get it done today. My commentary would be if the desired intent of changing the demo would be to put it in a file (which is honestly how I did it initially) then `/tmp/test.txt` would be a better option to target.

If I can replicate the error then it's worth the change in the demo guide. Just need to see if some change in bookworm made the output to stdout inoperable.

Reply:
Just tried it out now and am unable to replicate #47, so this is still unconfirmed as a needed change. Still going to hear out @m6n50u7 until the issue is resolved to their satisfaction or the tutorial is deemed satisfactory with obvious caveats that some installations may have configurations that necessitate departure from the tutorial to confirm functionality.